source/opt/debug_info_manager.cpp

```diff
@@ -449,12 +449,13 @@ bool DebugInfoManager::IsDeclareVisibleToInstr(Instruction* dbg_declare,
   return IsAncestorOfScope(instr_scope_id, decl_scope_id);
 }
 
-void DebugInfoManager::AddDebugValue(Instruction* scope_and_line,
+bool DebugInfoManager::AddDebugValue(Instruction* scope_and_line,
                                      uint32_t variable_id, uint32_t value_id,
```

Comment:
See my comment in #3461.  It's confusing that `AddDebugValue` doesn't always does what the caller meant (https://github.com/KhronosGroup/SPIRV-Tools/pull/3461#discussion_r456500705).